src/main/java/seedu/duke/ui/Ui.java

```diff
@@ -30,4 +33,18 @@ public static void commandList() {
     public static void printExitMessage() {
         System.out.println("So long buddy!");
     }
+
+    public static void printRouteMessage(ArrayList<Bus> options) {
+        int i = 1;
+        System.out.println("\nThe buses you can take are: ");
+        for (Bus option: options) {
+            if (option.toString() != null) {
+                System.out.println(i + ". " + option.toString());
+                i++;
+            }
+        }
+        if (i == 1) {
+            System.out.println("none.\n\tThere are no direct bus routes. :(");
+        }
+    }
```

Comment:
Might be better to separate i to a counter and a boolean for better clarity.

Reply:
Okie! I will follow that approach.